test/primitives/array-element-test.js

```diff
@@ -176,6 +196,18 @@ describe('ArrayElement', function() {
         arrayElement.push('foobar');
         itAddsToArray(arrayElement);
       });
+
+      it('sets the parent to children', function() {
+        var item = new minim.StringElement('foobar');
+        arrayElement.push(item);
+        expect(item.parents).to.include(arrayElement);
```

Comment:
Parent just provides access to the node's parents. This is useful in functions like `find` where you want to find certain elements that have a specific kind of parent. Also allows traversing up the tree.